Documentation/admin_guide.md

```diff
@@ -34,16 +34,16 @@ If `--wal-dir` flag is set, etcd will write the write ahead log files to the spe
 
 ### Cluster Management
```

Comment:
Good catch. I just missed that one.

To double check before I fix this,  the heading *Cluster Managment* belongs on the same level (level 2 -`##`) as *Data Directory*, correct?